scenes/ink_combat/ink_blob/ink_blob.gd

```diff
@@ -42,12 +42,17 @@ func _set_can_hit_enemy(new_can_hit_enemy: bool) -> void:
 func _ready() -> void:
 	var color: Color = INK_COLORS[ink_color_name]
 	visible_things.modulate = color
-	var impulse: Vector2 = direction.normalized() * SPEED
+	var impulse: Vector2 = direction.normalized() * speed
 	apply_impulse(impulse)
 
 
 func _process(_delta: float) -> void:
 	visible_things.rotation = linear_velocity.angle()
+	if node_to_follow:
+		var direction: Vector2 = node_to_follow.global_position - global_position
+		var force: Vector2 = direction.normalized() * speed
+		constant_force = Vector2(0, 0)
+		add_constant_central_force(force)
```

Comment:
I'm not completely sure this would work the same, but, wouldn't resetting the constant_force to 0 and then adding a constant central force be the same as just changing the constant_force to the new `force`?
Something like:

```suggestion
		constant_force = force
```